js/parts/Series.js

```diff
@@ -4192,6 +4196,7 @@ null,
         if (series.visible) {
             series.drawPoints();
         }
+        fireEvent(series, 'afterDrawPoints');
         /* series.points.forEach(function (point) {
```

Comment:
Do we need commented area?

Reply:
This piece of code is already inside a comment block on the master script.